[2024-08-07T00:01:01.003-0300] {processor.py:161} INFO - Started process (PID=10871) to work on /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-07T00:01:01.004-0300] {processor.py:830} INFO - Processing file /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-07T00:01:01.005-0300] {logging_mixin.py:188} INFO - [2024-08-07T00:01:01.005-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-07T00:01:01.008-0300] {logging_mixin.py:188} INFO - [2024-08-07T00:01:01.007-0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-07T00:01:01.008-0300] {logging_mixin.py:188} INFO - [2024-08-07T00:01:01.008-0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-07T00:01:01.008-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-07T00:01:01.017-0300] {processor.py:183} INFO - Processing /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-08-07T00:01:55.526-0300] {processor.py:161} INFO - Started process (PID=11156) to work on /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-07T00:01:55.530-0300] {processor.py:830} INFO - Processing file /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-07T00:01:55.532-0300] {logging_mixin.py:188} INFO - [2024-08-07T00:01:55.531-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-07T00:01:55.534-0300] {logging_mixin.py:188} INFO - [2024-08-07T00:01:55.534-0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-07T00:01:55.535-0300] {logging_mixin.py:188} INFO - [2024-08-07T00:01:55.535-0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-07T00:01:55.535-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-07T00:01:55.544-0300] {processor.py:183} INFO - Processing /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-08-07T00:02:44.051-0300] {processor.py:161} INFO - Started process (PID=11892) to work on /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-07T00:02:44.055-0300] {processor.py:830} INFO - Processing file /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-07T00:02:44.057-0300] {logging_mixin.py:188} INFO - [2024-08-07T00:02:44.056-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-07T00:02:44.060-0300] {logging_mixin.py:188} INFO - [2024-08-07T00:02:44.059-0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-07T00:02:44.060-0300] {logging_mixin.py:188} INFO - [2024-08-07T00:02:44.060-0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-07T00:02:44.060-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-07T00:02:44.071-0300] {processor.py:183} INFO - Processing /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-08-07T00:03:34.654-0300] {processor.py:161} INFO - Started process (PID=12141) to work on /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-07T00:03:34.658-0300] {processor.py:830} INFO - Processing file /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-08-07T00:03:34.659-0300] {logging_mixin.py:188} INFO - [2024-08-07T00:03:34.659-0300] {dagbag.py:545} INFO - Filling up the DagBag from /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-07T00:03:34.661-0300] {logging_mixin.py:188} INFO - [2024-08-07T00:03:34.661-0300] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-08-07T00:03:34.661-0300] {logging_mixin.py:188} INFO - [2024-08-07T00:03:34.661-0300] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-08-07T00:03:34.661-0300] {processor.py:842} WARNING - No viable dags retrieved from /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-08-07T00:03:34.739-0300] {processor.py:183} INFO - Processing /home/wilamy/PARA/1_projetos/Data_Lake_Football/Data_Lake_Football_Airflow/venv_airflow_data_lake/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.089 seconds
